internal/flink/command_endpoint.go

```diff
@@ -0,0 +1,34 @@
+package flink
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v4/pkg/cmd"
+)
+
+const (
+	publicFlinkEndpointType  = "PUBLIC"
+	privateFlinkEndpointType = "PRIVATE"
+)
+
+type flinkEndpointOut struct {
+	IsCurrent bool   `human:"Current" serialized:"is_current"`
+	Endpoint  string `human:"Endpoint" serialized:"endpoint"`
+	Cloud     string `human:"Cloud" serialized:"cloud"`
+	Region    string `human:"Region" serialized:"region"`
+	Type      string `human:"Type" serialized:"type"`
+}
+
+func (c *command) newEndpointCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:         "endpoint",
+		Short:       "Manage Flink endpoint.",
+		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireNonAPIKeyCloudLogin},
+	}
+
+	cmd.AddCommand(c.newEndpointListCommand())
+	cmd.AddCommand(c.newEndpointUseCommand())
+	cmd.AddCommand(c.newEndpointUnsetCommand())
```

Comment:
```suggestion
	cmd.AddCommand(c.newEndpointUnsetCommand())
	cmd.AddCommand(c.newEndpointUseCommand())
```
Nitpick: alphabetized order